cityofaustin/atd-product
# 10524 upgrade bootstrap v5

Fixes cityofaustin/atd-data-tech/issues/10524

This PR upgrades bootstrap to v5 and react-bootstrap to the corresponding v2.5. There were a lot of breaking changes with these upgrades but I think I combed through and fixed them all, let me know if there is something I missed! I got this iteration of the website looking super close to the original aside from the padding being a tiny bit off, I can't seem to get everything to align exactly as it was before but I think this is okay? We are also about to revamp the website so I figured I shouldn't waste too much time being a perfectionist if we will be changing a lot.

https://getbootstrap.com/docs/5.0/migration/
https://react-bootstrap.github.io/migrating/

Steps to test:

[Netlify preview](https://deploy-preview-49--atd-product.netlify.app/)

1. You can compare the netlify deploy to the current website to check if there are any big differences between them that need to be corrected.
2. Check the Home page, Our work, What we deliver, About us pages, along with the projects details, products details, click on all the tabs and filters, make sure everything is working/looks good!
